readme.html

```diff
@@ -35,7 +35,130 @@ <h2>Authentication</h2>
+      - groups:read
+      - im:read
+      - mpim:read
+      - users:read
+      - chat:write
+      - reactions:write
+      - files:write
+    bot:
+      - channels:manage
+      - groups:write
+      - im:write
+      - mpim:read
+      - mpim:write
+      - channels:read
+      - groups:read
+      - im:read
+      - users:read
+      - chat:write
+      - reactions:write
+      - files:write
+      - chat:write.public
+      - app_mentions:read
+      - channels:history
+      - im:history
+settings:
+  event_subscriptions:
+    bot_events:
+      - app_mention
+      - message.channels
+      - message.im
+  interactivity:
+    is_enabled: true
+  org_deploy_enabled: false
+  socket_mode_enabled: true
+  token_rotation_enabled: false
+</pre>
+<br><br>
+Select <b> From an app manifest </b> option.
+<br><br>
+<a href="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_select_from_app_manifest.png">
+  <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_select_from_app_manifest.png">
+</a>
+<br><br>
+Select a workspace.
+<br><br>
+<a href="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_select_workspace.png">
+      <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_select_workspace.png">
+</a>
+<br><br>
+Paste above manifest file in <b>yaml</b> section.
+<br><br>
+<a href="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_enter_app_manifest.png">
+      <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_enter_app_manifest.png">
+</a>
+<br><br>
+Once click on next, verify all scopes for Bot and User which is added from the manifest file.
+<br><br>
+<a href="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_review_summary_oauth.png">
+      <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_review_summary_oauth.png">
+</a>
+<br><br>
+Event subscriptions for the bot will be added from the manifest file, Click on create.
+<br><br>
+<a href="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_review_summary_settings.png">
+      <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_review_summary_settings.png">
+</a>
+<br><br>
+On the next page, there is a <b>Verification Token</b>. This token will be needed during asset configuration.
+<br><br>
+<a href="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_basic_info.png">
+  <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_basic_info.png">
+</a>
+<br><br>
+On the same page, there is a <b>App Level tokens</b>. This dialog box will have a <b>Generate Token and Scope</b>
+button, click on it. It would again open a pop up, give token name <b>socket_token</b>. Just below this would be the
+<b>Add Scope</b> button, add <b>connection:write & authorization:read</b> and click on <b>Generate</b>. This token will
+be needed during asset configuration.
```

Comment:
```suggestion
button. You will then be prompted to provide a token name <b>socket_token</b>. Just below this is the
<b>Add Scope</b> button. Add <b>connection:write & authorization:read</b> and click on <b>Generate</b>. This token is
needed during asset configuration.

Reply:
@aparekh-crest does the token name entered on for this prompt have to be "socket_token"?

Reply:
@aparekh-crest ok good to know. Then I would suggest revising the sections with references to "socket_token" to something like this

> button. You will then be prompted to provide a token name, for example, <b>socket_token</b>. Just below this is the

Reply:
@splunk-jessica Sounds good, will do that.